lib/resty/waf/request.lua

```diff
@@ -13,6 +13,11 @@ local table_insert = table.insert
 _M.version = base.version
 
 function _M.parse_request_body(waf, request_headers, collections)
+    if not waf._process_body then
+        --_LOG_"process_body option disabled, ignoring the body"
```

Comment:
Will this be super spammy?

Reply:
Oh, or maybe it's commented out. Derp.

Reply:
This is some convention this module has, the build process replaces these log comments with actual logging code and it'll only log when debug logging is enabled.